Files/Views/SettingsPages/Experimental.xaml

```diff
@@ -45,6 +46,16 @@
                     Header="Cache files and folders for better performance"
                     HeaderTemplate="{StaticResource CustomHeaderStyle}"
                     IsOn="{Binding UseFileListCache, Mode=TwoWay}" />
+
+                <controls:NumberBox 
+                    x:Uid="PreemptiveCacheParallelLimits"
+                    HorizontalAlignment="Left"
+                    Header="Preemptive cache parallel limit (0 turns off the preemptive cache)" 
+                    FontSize="17"
+                    SpinButtonPlacementMode="Inline"
+                    Width="300"
+                    Value="{Binding PreemptiveCacheParallelLimit, Mode=TwoWay}" 
+                    Minimum="0" />
```

Comment:
```suggestion
                    Minimum="0" 
                    HeaderTemplate="{StaticResource CustomHeaderStyle}"
                    IsEnabled="{Binding UseFileListCache, Mode=OneWay}"/>
```

Reply:
The issue with screen reader seems to be rooted in WinUI library. I didn't found such bug for NumberBox specificaly, but there are quite such bugs for other controls, like radio buttons: https://github.com/microsoft/microsoft-ui-xaml/issues/3183

I'm not quite sure what to do here

Reply:
If there was one person who would know, it would probably be @chingucoding 😂

Reply:
What exactly is the question here?

Reply:
@chingucoding for some reason the header of the number box isn't read by the screen reader.

Reply:
This is a known issue with NumberBox and the fact that it wraps around a TextBox which conflict with each other. See https://github.com/microsoft/microsoft-ui-xaml/issues/3505 and the fixing PR https://github.com/microsoft/microsoft-ui-xaml/pull/3775 for more context.

Reply:
Thanks! I guess this will be fixed in one of the next WinUI releases so it's nothing to worry about.